cfgov/jinja2/v1/browse-basic/index.html

```diff
@@ -13,8 +13,8 @@
     {% for block in page.header -%}
         {% if 'featured_content' in block.block_type %}
             <div class="block
-                        block__flush-sides
-                        block__border">
+                        block__border
+                        {{ 'u-mt0' if loop.index == 1 }}">
```

Comment:
This has always been an issue with the flush modifiers, agree this solution is probably best. Maybe leave a jinja comment on the next line so anyone looking at it in the future understands the reasoning.

Reply:
@jimmynotjim That's a good point -- comments added.